src/index.html

```diff
@@ -5,8 +5,436 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+
+      <div class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="logo">
```

Comment:
Let's fix it everywhere
```suggestion
        <img
          src="images/logo.png"
          alt="logo"
        >
```